sources/packages/backend/libs/utilities/src/integration-utils.ts

```diff
@@ -22,3 +22,25 @@ export function getTotalYearsOfStudy(completionYears: string): number {
       return 1;
   }
 }
+
+/**
+ * Filler to replace a string with default value, when given value is
+ * null or empty.
+ * @param value provided value.
+ * @param defaultValue default value to replace.
+ * @returns value replaced with filler if required.
+ */
+export function emptyStringFiller(value: string, defaultValue = ""): string {
```

Comment:
We can extend the StringBuilder class to something like "SpecializedStringBuilder" and we can add new methods like `appendSringWithFiller` and `appendNumberWithFiller` where the new class could also receive in the constructor the `NUMBER_FILLER` and `STRING_FILLER` which allow u to stop providing then on every append call.